pandas/tests/generic/test_to_xarray.py

```diff
@@ -106,19 +92,14 @@ def test_to_xarray(self):
 
 
 class TestSeriesToXArray:
-    @pytest.mark.skipif(
-        not _XARRAY_INSTALLED
-        or _XARRAY_INSTALLED
-        and LooseVersion(xarray.__version__) < LooseVersion("0.10.0"),
-        reason="xarray >= 0.10.0 required",
-    )
+    @td.skip_if_no("xarray", "0.10.0")
     def test_to_xarray_index_types(self, indices):
         if isinstance(indices, pd.MultiIndex):
             pytest.skip("MultiIndex is tested separately")
 
         from xarray import DataArray
 
-        s = Series(range(len(indices)), index=indices)
+        s = Series(range(len(indices)), index=indices, dtype="float64")
```

Comment:
why is this change needed?

Reply:
To suppress the Deprecation warning

<details>

```DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.```

</details>

Reply:
Is this the only place this occurs? I assume we are getting an empty container through from indices

I'd be surprised if this is the only case, so not sure doing as a one-off here

Reply:
> Is this the only place this occurs? I assume we are getting an empty container through from indices
> 
> I'd be surprised if this is the only case, so not sure doing as a one-off here

@WillAyd This is the only case in this file that raises the deprecation warning.

Reply:
I can revert this change, not a deal breaker for me